tests/organic_queries/check_organic_queries.py

```diff
@@ -0,0 +1,154 @@
+# ruff: noqa: E402
+import bittensor as bt
```

Comment:
Will this test be able to run in the CI/CD pipeline?

Reply:
No, testing files that are run by pytest need to start with test_. So this is a test to run manually